### PR TITLE
[MIG] pos_operating_unit: Migration to 17.0

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: pos_operating_unit/views/pos_views.xml [hunk 1]
```diff
@@ -5,22 +5,19 @@
...
-                            />
-                            </div>
-                        </div>
-                    </div>
+            <xpath expr="//setting[@id='other_devices']" position="before">
+                <setting
+                    id="operating_units_setting"
+                    string="Operating Units"
+                    help="Operating units used for this POS"
+                >
```
Comment (traitrantrobz) [inline pos_operating_unit/views/pos_views.xml:13]: @jdidderen-nsi Why remove groups?

**Comment by traitrantrobz** [comment]
Exception [review](https://github.com/OCA/operating-unit/pull/693#discussion_r1666542347), everything else is LGTM!

File: pos_operating_unit/views/pos_views.xml (hunk 1 above)
Comment (ghost) [reply pos_operating_unit/views/pos_views.xml:13]: For no reason, a simple mistake when copying example from standard Odoo. 
I added the groups on every operating_unit_id(s) field to fit with other OU modules

**Review by yvaucher — APPROVED** [review]
LGTM

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**